bindings/python/dlite-python.i

```diff
@@ -874,24 +876,28 @@ int dlite_swig_set_scalar(void *ptr, DLiteType type, size_t size, obj_t *obj)
         dest->description = (src->description) ? strdup(src->description) :NULL;
 
       } else if (PySequence_Check(obj) && PySequence_Length(obj) == 6) {
+        /* We should deprecate this - either remove or change to dict... */
```

Comment:
Why?

Reply:
Because I feel it is too easy to make unnoticed mistakes if we allow to set a property sequence of the form:

```python
(name, type, ref, shape, unit, descr)
```

It will also be susepatble to errors if we in the future make any changes to the property arguments.

It think it will be more explicit and robust to instead require a dict of the form

```python
{"name": name, "type": type, "ref": ref, "unit": unit, "description": descr}
```